### ISSUE TITLE
Rooms don't show as unread when threads which I participate in have new messages

### ISSUE THREAD

**Issue by turt2live** [element-hq/element-web#27470]
### Steps to reproduce

You will need 2 users, including yourself.

1. As your opponent, send a message.
2. As you, create a thread from that message. Click away to another room after.
3. As your opponent, send a message on that thread.
4. As you, note the room doesn't appear as unread (but the message preview updates)

#### What did you expect?

The room should be marked as unread: this is a thread I care about and should receive blocking notifications for.

#### What happened instead?

Nothing :D

### Operating system

Windows 11

### Application version

Element Nightly version: 0.0.1-nightly.2024051401 Crypto version: Rust SDK 0.7.0 (61b175b), Vodozemac 0.5.1

### How did you install the app?

The Internet

### Homeserver

t2l.io

### Will you send logs?

Yes

**Comment by wu-lee** [comment]
I think I'm seeing this too. It mostly seems to relate to threaded messages, not yet certain if it occurs for unthreaded ones. 
- No unread dot next to rooms with unread messages, but
- A dot appears next to unread replies collapsed in the room panel
- Threads show as unread in the threads panel to the right of the room
- Notifications still seem be seen by the browser when messages arrive, as these pop upon the desktop.

Element version: 1.11.66
Crypto version: Olm 3.2.15
Local Matrix server is Synapse 1.107.0
My browser is Firefox (LibreWolf) 125.0.3-1 on Linux. Other users using the same client and server have been reporting this, however.

Installed via Cloudron, which reports:
- Element 1.11.66
  - App ID: 2f12a71c-4f4b-4cde-a715-d3e2ba6ae7be
  - Package Version: [im.riot.cloudronapp@1.9.24](https://my.digitalcommons.coop/#/appstore/im.riot.cloudronapp?version=1.9.24)
  - Last Updated: 05/08/2024
- Matrix Synapse 1.107.0
  - App ID: e78b175e-95ab-4466-a4a2-2f4ce723a82a
  - Package Version: [org.matrix.synapse@1.93.0](https://my.digitalcommons.coop/#/appstore/org.matrix.synapse?version=1.93.0)
  - Last Updated: 05/15/2024

**Comment by langleyd** [comment]
@turt2live thread activity has been removed from the room list to as it made the room list too noisey. So we would no longer expect it there.

The thread activity center can be used to get your thread notifications(bottom left, above settings). By default this will only give thread notifications(green and red dots), but if you want thread activity in here you can enabled it from notification settings(turn off the "Only show notifications in the thread activity centre" toggle).

I don't believe there is currently a way for us to track "this is a thread I care about". We do however want to add thread participation back into the solution in future which should accommodate this use case.

Please feel free to re-open if I have misunderstood something.

**Comment by turt2live** [comment]
I'd like to keep this open as a feature request, as currently it's leading to me missing some extremely important messages. 

What are green dots, and what is considered "thread activity"? As a user, my participating threads feel like they should be 'notifications'.

**Comment by t3chguy** [comment]
Lowering O due to most users seemingly preferring the TAC experience

**Comment by turt2live** [comment]
Do they prefer the experience, or are they unaware of the bug/feature? 😇

**Comment by t3chguy** [comment]
I have seen comments saying that it fixed threads and stuck notifications for them

**Comment by wu-lee** [comment]
Essentially all our users were at first unaware of the change in the notification UI, then alarmed at the number of messages they had been missing. (Including me!  I assumed this was a bug.) This despite the new "thread activity centre", which is there but very easy to miss, and its intention not clear.

So this is us saying that the notifications as they were before were more intuitive, and helpful not "noisy", at least when they worked correctly. (Based on comments I've seen, general confidence in the reliability of Element's notifications has been dropping, and this has reduced it further.)   Switching this without any warning to users - who may be unaware there has been an upgrade - adds to the problem.

Perhaps the new "thread activity centre" should be an option rather than a replacement?

**Comment by wu-lee** [comment]
(I should add, a big reason users are comfortable with the unread message markers next to channels is that this is how it works currently on What's App, Telegram, Signal and Slack - so if not all the other similar applications, at least the widely used ones)

**Comment by t3chguy** [comment]
> Essentially all our users were at first unaware of the change in the notification UI

> which is there but very easy to miss, and its intention not clear.

<img width="1705" alt="image" src="https://github.com/element-hq/element-web/assets/2403652/c14da9e7-e956-404b-921a-ceabb482b500">

Did you not see the in-app release announcement?

**Comment by turt2live** [comment]
I never got that announcement, fwiw

**Comment by wu-lee** [comment]
No, I don't remembers seeing that. I infer no-one else on our instance did either?  Perhaps it appeared and then got dismissed without being noticed or fully comprehended.

**Comment by turt2live** [comment]
I'm still missing important messages in threads, even with TAC. People are having to reach out to me via DM to get me to see their messages.

**Comment by TobiasNx** [comment]
We were migrating from rocket.chat to element, showing unread messages of threads in their hosting channel/room is a feature we are dearly missing which rocket.chat supports.

The Thread Activity Center is not very helpful since it is hidden and you need extra steps to find unread messages in room-threads.

**Comment by langleyd** [comment]
👍 Thread Activity Center is not a final solution, more of stop-gap. We would like to build out a more fully featured Activity Center and add better tools for thread participation, see threads inline, etc. It's likely we will make fundamental improvements to the timeline first though, so we probably won't get to Activity Center/Thread Participation before say Q42025.

**Comment by knarrff** [comment]
I already regarded this as a bug, and I can't quite wrap my head around this being intentional. To me, threads are just a way to keep messages in a room somewhat sorted - in a similar way email threads are working. As such, messages in a thread still belong to that room, so I am on the same page as lots of others apparently are: unread messages in threads should mark the room the thread belongs to as 'unread'.

**Comment by mkmelin** [comment]
I missed so many messages before I found the setting to turn this off.
